docs/sources/installation/sizing/index.md

```diff
@@ -42,6 +42,21 @@ This tool helps to generate a Helm Charts `values.yaml` file based on specified
   </div>
 
   <a v-bind:href="helmURL" class="primary-button">Generate and download values file</a>
+
+  <blockquote v-if="help">
+    <span v-if="help === 'ingest'">
+    Defines the log volume in terrabytes expected to be ingested each day.
+    </span>
+    <span v-else-if="help === 'node'">
+    Defines the node type of the Kubernetes cluster.
+    </span>
+    <span v-else-if="help === 'retention'">
+    Defines how long the ingested logs should be kept.
+    </span>
+    <span v-else-if="help === 'queryperf'">
+    Defines the expected query performance. Basic is... Super is...
```

Comment:
"Basic is... Super is..."  Was there going to be more text here?

Reply:
Yes. I need @trevorwhitney to explain when to use Basic and when to use Super.

Reply:
Basic is ~3mbps, Super is better than that (I don't have exact numbers, we would need to run tests to get them)

Reply:
@JStickler could you take another look :slightly_smiling_face: